web/frontend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "HTTPS=false react-scripts start",
+    "start": "HTTPS=true react-scripts start",
```

Comment:
does it still work with the mocked backend ?

Reply:
yes, I tried both and it works